packages/material/src/layouts/layout.util.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+    DispatchRenderer,
+    JsonSchema,
+    UISchemaElement,
+  } from '@jsonforms/core';
+import Grid from 'material-ui/Grid';
+
+const renderChildren = (
+    elements: UISchemaElement[],
+    schema: JsonSchema,
+    path: string
+  ) => {
+
+    return elements.map((child, index) =>
+    (
+        <Grid item key={`${path}-${index}`} xs>
+          <DispatchRenderer
+            uischema={child}
+            schema={schema}
+            path={path}
+          />
+        </Grid>
+      )
+    );
+  };
+export interface MaterialLayoutRendererProps {
+    elements: UISchemaElement[];
+    schema: JsonSchema;
+    path: string;
+    visible: boolean;
+    direction: 'row'|'column';
+}
+export const MaterialLayoutRenderer = (
+    {visible, elements, schema, path, direction}: MaterialLayoutRendererProps) => {
+        if (!_.isEmpty(elements)) {
```

Comment:
This will cause an error if `elements` is empty. Return `null` if you don't want to render anything instead